index.html

```diff
@@ -1,248 +1,308 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
+    <!-- External Css -->
     <link rel="stylesheet" href="assets/css/style.css">
-    <!-- Poppins font form Google APIs cdn -->
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
+
 <body>
-    <header id="header">
-        <a href="#" id="my_logo">My Logo</a> 
+  <div class="wrap">
+    <header class="toolbar">
+      <div class="container">
+        <a href="#"> My Logo </a>
         <nav>
-            <a href="#" id="img_link">
-                <img src="assets/images/menu.png" id="menu_img" alt="menu">
-            </a>
-        </nav>                
+          <a href="#">
+            <img src="assets/images/menu.png" alt="" />
+          </a>
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
     </header>
```

Comment:
- [ ] Good job designing the mobile version of your portfolio :clap: , however I noticed that all over the sections you use a font that does not stick to the Figma design, Kindly follow the [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=mzXac1hAq7vplnjR-0) to the Figma design and update the part in red to stick to the Figma design :100: .Always see requirements as client need you need to meet :heavy_check_mark: 

![Screenshot from 2023-01-21 12-35-30](https://user-images.githubusercontent.com/80458199/213863595-b05b62b3-3990-4acf-8f2d-adea1f365c15.png)


![Screenshot from 2023-01-21 12-36-08](https://user-images.githubusercontent.com/80458199/213863597-737b55d9-5333-4669-ad01-38fac4af242e.png)

![Screenshot from 2023-01-21 12-36-24](https://user-images.githubusercontent.com/80458199/213863600-14096187-11a6-44fe-8ef2-3ce1c894fabf.png)